tiatoolbox/models/architecture/hovernet.py

```diff
@@ -277,6 +371,53 @@ def forward(self, prev_feat: torch.Tensor):
         return feat
 
 
+def create_decoder_branch(out_ch=2, ksize=5):
+    """Helper to create a decoder branch."""
+    modules = [
```

Comment:
Rather than refactoring it out into a function. It is better to put it as a private static_method to explicitly specify that this decoder creation is pertained to HoVerNet family

Reply:
Changed to static protected method as part of HoVer-Net as discussed.